### ISSUE TITLE
Unrecognized Line Breaks when passing code to Terminal in Windows

### ISSUE THREAD

**Issue by EconometricsBySimulation** [julia-vscode/julia-vscode#826]
I don't know if this is really a issue with `Julia-vscode` since in windows default consoles I am getting the same issue, but I was hoping that perhaps it was something that `Julia-vscode` could address.

The following is what happens when I pass code from editor to console [Ctr-Enter]. I tried adjusting the LF/CRLF on the bottom of the screen but it does not seem to do anything. 

I guess I would not care if it was purely a cosmetic issue. But passing that particular piece of code to Julia takes about 30 seconds because the console seems confused as to what to do with it.

Any advice?

```
julia> function rangeformatter(txtin; rangeformat, seperator = "through|till| ", toomany = 30)
           txt = replace(txtin, r"[|./\\-]" => " ")
               txt = replace(txt, r"\b([0-9])\b" => s"0\1") |> x->replace(x, r"[ ]+" => " ")
                 # Input range formats
                     rf1 = rangeformat[1]
                         if length(rangeformat) == 1 ; rf2 = rangeformat[1]; end
                             if length(rangeformat) >= 2 ; rf2 = rangeformat[2]; end
                                 left, right = dateformat2regex.([rf1,rf2])
                                     rangematch = match(Regex("^($left)([ ]*)($seperator)([ ]*)($right)"), txt)
                                         (rangematch === nothing) && (return [])
                                             m1 = rangematch.captures[1]
                                                 m2 = rangematch.captures[5]
                                                   # Interprettable range formats
                                                       if length(rangeformat) == 4 ; rf1 = rangeformat[4]; rf2 = rangeformat[4]; end
                                                           if length(rangeformat) == 5 ; rf1 = rangeformat[4]; rf2 = rangeformat[5]; end
                                                               if length(rangeformat) >= 3 ; m1, m2 = rmwhitespace.(rangeformat[3](m1, m2)); end
                                                                   try
                                                                           dt1 = Date(m1, DateFormat(rf1))
                                                                                   dt2 = Date(m2, DateFormat(rf2))
                                                                                       # Define a place to start capturing the text input after the date
                                                                                               starter = length(rangematch.match) + (length(txtin) - length(txt)) + 1
                                                                                                       txtout = strip(txtin[starter:end])
                                                                                                           # txtout = replace(txtin, Regex("^($left)([ ]*)($seperator)([ ]*)($right)")=>"")
                                                                                                                   indt   = txtin[1:(starter - 1)]
                                                                                                                       # indt   = rangematch.match
                                                                                                                               (length(dt1:Day(1):dt2) > toomany) && return []
                                                                                                                                       return DataFrame(date = collect(dt1:Day(1):dt2),
                                                                                                                                                      txt = fill(txtout, length(dt1:Day(1):dt2)),
                                                                                                                                                                     indt = fill(indt, length(dt1:Day(1):dt2)))
                                                                                                                                                                         catch
                                                                                                                                                                                 return  []
                                                                                                                                                                                     end
                                                                                                                                                                                     end
rangeformatter (generic function with 1 method)
```

**Comment by davidanthoff** [comment]
I think the Julia REPL is inserting indentation there, which we of course don't want when we paste code with Ctrl+Enter. Not sure what can be done about that...

You might try Alt+Enter to run your code, that uses a different way of running code that should work better for large amounts of code.

**Comment by KristofferC** [comment]
Using the bracket pasting code (which I once PRed) should fix this. Not sure if that got removed or not used in this case.

**Comment by davidanthoff** [comment]
We removed the bracket pasting stuff on Windows because it didn't work there. I don't remember whether the problem was that the Windows console infrastructure didn't support it, or that the libuv fork used by Julia is not using the right Windows APIs to support this, or something else.

**Comment by KristofferC** [comment]
The bracket paste is interpreted by the Julia REPL so not sure why there would be a OS difference.

**Comment by davidanthoff** [comment]
I just tried to reenable the bracket pasting stuff that @KristofferC had put in, and it seemed to work now. But the indentation only seemed correct if I configured VS Code to _not_ use the new conpty API (in which case I think it falls back to winpty). So it seems certainly worth investigating again whether we can just enable it on Windows.

We probably have to test on different Windows versions, though... When I took it out, I was running an older version of Windows 10, and in the meantime the whole console infrastructure in Windows 10 and VS Code has changed a lot. So maybe it didn't work on older Windows versions, but now does... Or maybe it was the VS Code version, or who knows.

**Comment by KristofferC** [comment]
Why it is surprising is that it is up to the running application to interpret the escape sequence being sent to it and the Julia REPL does this in a OS-agnostic manner:

https://github.com/JuliaLang/julia/blob/d245df961df863d12e5b41604988650246941537/stdlib/REPL/src/LineEdit.jl#L1886-L1891

**Comment by davidanthoff** [comment]
Actually, I might have reverted this for the wrong reason: I am positive that something didn't work and removing this fixed it, _but_ it might have been that bracket paste mode just doesn't work with a single line of text. That bug I fixed _after_ I had removed bracket paste for Windows. So it might well be that bracket paste mode, only used for multi line text, would have always worked on Windows.

In any case, I'm going to enable bracket paste on Windows across the board now again, it seems to work with the current version of VS Code and Windows.
